test/testqobj.jl

```diff
@@ -50,12 +51,15 @@ end
     @test ins isa Array{Any,1}
     for j in ins
         @test j isa Dict{String,Any} 
-        # test individual blocks here 
     end
+    c = build_circuit(4, 1, [1=>2, 2=>3, 3=>4])
+    inst = generate_inst(c)
+    c2 = inst |> inst2qbir
+    @test operator_fidelity(c, c2) ≈ 1
 end
 
 @testset "Variational Quantum Eigen Solver" begin
-    for n in 3:10
+    for n in [3, 6, 7, 8, 9]
         circuit = dispatch!(variational_circuit(n, n+1),:random)
```

Comment:
We should test different gates one by one. Rather than testing circuits as a whole.

Reply:
Yeah. I'll add those soon

Reply:
perfect!